django_rq/views.py

```diff
@@ -58,6 +59,28 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
+def _collect_workers_per_configuration(queue_list):
+    """Collects, into a list, dictionaries of connections_config and its
+    workers
+    """
+    workers_collections = []
+    for item in queue_list:
+        if item['connection_config'] not in [c['config'] for c in workers_collections]:
```

Comment:
We should call `filter_connection_params` on each `connection_config` to make this even more efficient. Queues may have different configs although they point to the same Redis connection. For example:
```python
RQ_QUEUES = {
    'default': {
        'HOST': 'localhost',
        'PORT': 6379,
        'DB': 0,
        'PASSWORD': 'some-password',
        'DEFAULT_TIMEOUT': 360,
    },
     'low': {
        'HOST': 'localhost',
        'PORT': 6379,
        'DB': 0,
        'PASSWORD': 'some-password',
        'DEFAULT_TIMEOUT': 60, # This timeout causes the connection_configs to be different even though they use the same Redis DB
    },
}```